apps/api/v2/src/ee/event-types/event-types_2024_06_14/controllers/event-types-private-links.controller.ts

```diff
@@ -0,0 +1,131 @@
+import { PrivateLinksService_2024_06_14 } from "@/ee/event-types/event-types_2024_06_14/services/private-links.service";
+import { VERSION_2024_06_14_VALUE } from "@/lib/api-versions";
+import { API_KEY_OR_ACCESS_TOKEN_HEADER } from "@/lib/docs/headers";
+import { GetUser } from "@/modules/auth/decorators/get-user/get-user.decorator";
+import { Permissions } from "@/modules/auth/decorators/permissions/permissions.decorator";
+import { ApiAuthGuard } from "@/modules/auth/guards/api-auth/api-auth.guard";
+import { PermissionsGuard } from "@/modules/auth/guards/permissions/permissions.guard";
+import {
+  Controller,
+  UseGuards,
+  Get,
+  Param,
+  Post,
+  Body,
+  Patch,
+  Delete,
+  ParseIntPipe,
+} from "@nestjs/common";
+import { ApiHeader, ApiOperation, ApiTags as DocsTags } from "@nestjs/swagger";
+
+import {
+  EVENT_TYPE_READ,
+  EVENT_TYPE_WRITE,
+  SUCCESS_STATUS,
+  VERSION_2024_06_14,
+} from "@calcom/platform-constants";
+import {
+  CreatePrivateLinkInput_2024_06_14,
+  UpdatePrivateLinkInput_2024_06_14,
+  CreatePrivateLinkOutput_2024_06_14,
+  GetPrivateLinksOutput_2024_06_14,
+  UpdatePrivateLinkOutput_2024_06_14,
+  DeletePrivateLinkOutput_2024_06_14,
+  TimeBasedPrivateLinkOutput_2024_06_14,
+  UsageBasedPrivateLinkOutput_2024_06_14,
+} from "@calcom/platform-types";
+
+@Controller({
+  path: "/v2/event-types/:eventTypeId/private-links",
+  version: VERSION_2024_06_14_VALUE,
```

Comment:
Since there is only 1 version of event types private links endpoints there is no reason to version it - you can move all the new code out of `event-types_2024_06_14` folder to `v2/src/ee/event-types-private-links` and remove `version` from "@Controller" and all the `_2024_06_14` suffixes from controller and services etc.